docs/reference/templates.rst

```diff
@@ -201,13 +202,21 @@ a global custom template and then override that customization on a specific
 
 Finding configured templates
 ----------------------------
-Templates that are set using the ``setTemplate()`` or ``setTemplates()``
-methods can be accessed through the ``getTemplate($name)`` method of an
-Admin.
+Each ``Admin`` has a ``TemplateRegistry`` service connected to it that holds
+the templates registered through the configuration above. Through the method
+``getTemplate($name)`` of that class, you can access the templates set for
+that ``Admin``. The ``TemplateRegistry`` is available through ``$this->templateRegistry``
+within the ``Admin``. Using the service container the template registries can
+be accessed outside an ``Admin``. Use the ``Admin`` code + ".template_registry"
```

Comment:
Use backticks for `template_registry` too please.

Reply:
Around the quotes or instead of the quotes?

Reply:
:thinking: instead